cg/services/orders/order_status_service/order_summary_service.py

```diff
@@ -0,0 +1,40 @@
+from cg.apps.tb.api import TrailblazerAPI
+from cg.apps.tb.dto.summary_response import AnalysisSummary
+from cg.services.orders.order_status_service.dto.case_summary import CaseSummary
+from cg.services.orders.order_status_service.dto.order_summary import OrderSummary
+from cg.services.orders.order_status_service.utils import create_summaries
+from cg.store.models import Order
+from cg.store.store import Store
+
+
+class OrderSummaryService:
+    def __init__(self, analysis_client: TrailblazerAPI, store: Store) -> None:
+        self.analysis_client = analysis_client
+        self.store = store
+
+    def get_summary(self, order_id: int) -> OrderSummary:
+        return self.get_summaries([order_id])[0]
```

Comment:
I like the re-use of `get_summaries`. Do we need a sanity check to make sure the return of `get_summaries` is length 1?

Reply:
Yeah, I added it as a sanity check :+1: